src/etapi/attributes.js

```diff
@@ -40,7 +41,9 @@ function register(router) {
     });
 
     const ALLOWED_PROPERTIES_FOR_PATCH = {
-        'value': [v.notNull, v.isString]
+        'value': [v.notNull, v.isString],
+        'isInheritable': [v.notNull, v.isBoolean],
```

Comment:
`isInheritable` should be removed from PATCH. If you need to change this flag, you need to delete the attribute and create a new one with the desired value.

attribute type, name and isInheritable are considered immutable after creation and a lot of code relies on that.

Reply:
Fixed. Thanks for the background, that requirement will be pretty easy to accommodate.